README.md

```diff
@@ -18,6 +18,11 @@ Install the Python module (Python 3 only)
 
     pip3 install sacrebleu
 
+In order to install Japanese tokenizer support through `mecab-python3`, you need to run the
+following command instead, to perform a full installation with dependencies:
+
+    pip3 install sacrebleu[ja]
+
 Alternately, you can install from the source:
```

Comment:
Can you explain this? What does the `[ja]` signify? Is this a bash wildcard?

Reply:
This seems to be a good explanation:
https://stackoverflow.com/a/46775606